Upgrade NuGet package reference

epsitec opened the pull request.

epsitec committed:
Upgrade NuGet package reference

epsitec committed:
Update NuGet packages

epsitec committed:
Simplify project, keep netstandard2.1

epsitec committed:
Fix <TargetFramwork>

epsitec committed:
Merge branch 'vNext' into epsitec

epsitec committed:
Update nuget package

epsitec committed:
zou-flow: merge 'temp' into 'master' (8382167 -> 17b7d18)

# Conflicts:
#	OpenXmlPowerTools/OpenXmlPowerTools.csproj

ThomasBarnekow commented:
@epsitec, could you please explain what you want to do? You have a larger number of commits that only seem to change a single file (the .csproj file). The change itself does not seem to be acceptable, e.g., because you are only targeting `netstandard2.1`, which would leave all projects behind that are targeting any version of .NET Framework.

Commit:
Handle invalid twips values

CGuidi committed:
zou-flow: merge 'origin/vNext' into 'vNext' (c351f78 -> c2dabce)

epsitec committed:
Merge branch 'OpenXmlDev:vNext' into vNext

epsitec committed:
Upgrade NuGet and switch to net6

Commit:
[fix] Revert changes from net6 to netstandard2.1
Net6 discontinued support for the library System.Drawing.Common
Therefor the project cannot be deployed in net6 yet.
Going to net6 requires someintegration work:
- integrating third party drawing library (SkiaSharp for example)
- fixing CA1051 in a couple of files

CGuidi committed:
Merge pull request #1 from Grisou13/vNext

[fix] Revert changes from net6 to netstandard2.1

CGuidi committed:
zou-flow: merge 'origin/vNext' into 'vNext' (69800dc -> a39a767)

epsitec committed:
Update NuGet packages

CGuidi committed:
zou-flow: merge 'origin/vNext' into 'vNext'

# zou-flow - 0.5.0.2241 net6

Commit:
When assembling cell borders, direct formatting takes precedence

Commit:
If a row has tblPrEx properties, and if that has borders, then remove borders from what what propagated from the table

Commit:
Rationalize left/right borders of cells

Commit:
Make more robust

Commit:
Enable direct formatting on cells

Commit:
Fix null ref

rvuistin committed:
Fix/disable some warnings

rvuistin committed:
Merge branch 'vNext'

# Conflicts:
#	OpenXmlPowerTools/OpenXmlPowerTools.csproj

CGuidi committed:
zou-flow: bump version to 'v4.6.0'.

# zou-flow - 6.0.0.2334 net7

epsitec committed:
Merge branch 'OpenXmlDev:vNext' into vNext

epsitec committed:
Merge remote-tracking branch 'origin/vNext'

epsitec committed:
Upgrade to latest compatible OpenXml nuget package

epsitec commented:
Sorry for the noise. We will continue to use code from origin/vNext and pull/merge changes into our fork, rather than trying to push our changes for the public consumption. Our needs are specific to our build infrastructure and it would not make sense to have the official `OpenXmlPowerTools` projet integrate our changes.
